env.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Recreate config file
+rm -rf ./env-config.js
+touch ./env-config.js
+
+# Add assignment
+echo "window._env_ = {" >> ./env-config.js
+
+# Loop on environment variables prefixed with
+# io_onboarding_pa and add them to env-config.js
+for io_onboarding_pa_var in $(env | grep -i io_onboarding_pa); do
```

Comment:
just a remark that I don't like this approach much, I'd prefer a whitelist of variables as just enumerating all env variables starting with this prefix can be dangerous (i.e. passwords leaking out) - perhaps making it clear that the variable will be publicly exposed using `public_io_onboarding_pa` prefix?